components/public-api/gitpod/experimental/v1/teams.proto

```diff
@@ -41,6 +41,9 @@ message TeamMember {
 
     // primary_email is the primary email of the TeamMember
     string primary_email = 6;
+
+    // is_owned_by_organization indicates if the TeamMember is owned by the organization
+    bool is_owned_by_organization = 7;
```

Comment:
We don't have a strict standard here, but some guides suggest that the `is_` prefix is omitted, as it's presented by the type - https://google.aip.dev/140

```suggestion
    bool owned_by_organization = 7;
```

Reply:
I prefer the `is` prefix as it reads better in code and also doesn't require to look up the type. In the case, e.g., leaving it out would make think it is possibly of type `Organization`.

Reply:
Sure, there's a pre-existing pattern which doens't use the `is` - https://github.com/gitpod-io/gitpod/blob/main/components/public-api/gitpod/experimental/v1/projects.proto#L50

Reply:
ok, I don't care enough to discuss this further